lib/AppInfo/Application.php

```diff
@@ -120,12 +120,18 @@ public function checkAndEnableCODEServer() {
 			$appConfig->setAppValue('wopi_url', $new_wopi_url);
 			$appConfig->setAppValue('disable_certificate_verification', 'yes');
 
+			/** @var DiscoveryService $discoveryService */
 			$discoveryService = $this->getContainer()->get(DiscoveryService::class);
+			/** @var CapabilitiesService $capabilitiesService */
 			$capabilitiesService = $this->getContainer()->get(CapabilitiesService::class);
 
 			$discoveryService->resetCache();
 			$capabilitiesService->resetCache();
-			$capabilitiesService->fetchFromRemote();
+			try {
+				$capabilitiesService->fetch();
+				$discoveryService->fetch();
+			} catch (\Exception $e) {
```

Comment:
Do we need to do anything here with the caught error? Is it OK to catch it and just throw it away?